app/lib/mapIAPTResults.js

```diff
@@ -6,19 +6,19 @@ function mapWebsite(input) {
   const contacts = JSON.parse(input.Contacts);
   if (contacts) {
     const websiteContact = contacts.find(
-      contact => contact.OrganisationContactMethodType
+      (contact) => contact.OrganisationContactMethodType
         === constants.websiteContactMethodType
     );
 
     if (websiteContact) {
-      // eslint-disable-next-line no-param-reassign
+      // eslint-disable-next-line no-param-reassign, prefer-destructuring
```

Comment:
What is reason for not using destructuring here, and similar places where modifying an existing object?
Certainly to me the destructuring version is harder to read, but perhaps that's due to unfamiliarity.
`({ OrganisationContactValue: input.website } = websiteContact);`

Reply:
Agree on the harder to read sentiment. The setting of the rule options has been a bit overzealous. Another change is being made to the config at which I'll update this. I'll label the PR as WIP until that has been done.